packages/app/src/cli/services/versions-list.ts

```diff
@@ -0,0 +1,123 @@
+import {fetchOrCreateOrganizationApp} from './context.js'
+import {AppDeploymentsQuery, AppDeploymentsQuerySchema} from '../api/graphql/get_versions_list.js'
+import {AppInterface} from '../models/app/app.js'
+import {getAppIdentifiers} from '../models/app/identifiers.js'
+import {partnersRequest} from '@shopify/cli-kit/node/api/partners'
+import {ensureAuthenticatedPartners} from '@shopify/cli-kit/node/session'
+import {renderTable} from '@shopify/cli-kit/node/ui'
+import colors from '@shopify/cli-kit/node/colors'
+import {outputContent, outputInfo, outputToken, unstyled} from '@shopify/cli-kit/node/output'
+
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+type DeploymentLine = {
+  createdAt: string
+  createdBy?: string
+  message?: string
+  versionTag: string
+  status: string
+}
+
+function formatDate(date: Date) {
```

Comment:
could we extract this method to the `cli-kit` `string` utility module?, I mean this kind of date format could be really useful for devs